src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -300,6 +308,16 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
   }
 
 }
+function displayTipMessage(actions: any, selectedQuery: any) {
+  if (selectedQuery.tip) {
```

Comment:
The functions calling displayTipMessage already check if the tip exists, is there a need to double check?